gobblin-restli/server.gradle

```diff
@@ -37,7 +37,9 @@ ext.apiProject = project(project.getParent().getPath() + ':api')
 
 dependencies {
   compile project(path: project.getParent().getPath() + ":api", configuration: 'dataTemplate')
+  compile project(path: project.getParent().getPath() + ":client")
   compile project(':gobblin-utility')
+  compile project(':gobblin-runtime')
```

Comment:
This should not be a default dependency. Let's do the extra dependencies thing we talked about.